### ISSUE TITLE
Carousel does not work

### ISSUE THREAD

**Issue by miketricking** [angular-ui/bootstrap#6112]
### Bug description:
In your documentation if you copy or click the link to edit in plnkr it does not work and just displays all the images not in a slider at all

### Link to minimally-working plunker that reproduces the issue:
If you click the edit in plnkr button on your site (https://angular-ui.github.io/bootstrap/) you will see it does not work: http://plnkr.co/edit/YXMBHI6AqcF2EAUCud40?p=preview

### Version of Angular, UIBS, and Bootstrap

Angular: 1.5.5

UIBS: 2.0.0

Bootstrap: 3.3.6

**Comment by pkozlowski-opensource** [comment]
For me the http://plnkr.co/edit/YXMBHI6AqcF2EAUCud40?p=preview works perfectly fine. You need to share more info (OS, browser, screenshot demonstrating a problem). 

PS No need to "scream" with big bold letters...

**Comment by wesleycho** [comment]
I'm guessing the bold is due to not using a new line, and lack of understanding of markdown, not so much screaming :) .

**Comment by thegreatmichael** [comment]
I also had a problem with the carousel. My problem stemmed from my slides ids being 1, 2, and 3 (i was testing with 3 slides), but i set `active` to 0 (thinking it was an array index on the `$scope.slides` array, when in fact it is the `id` of the slide). I needed to set active to 1 to make it work... Actually, even when I did this, I still ended up with a problem. It looks like the ids need to start at 0?
